File: 1.FDP/2° Competencia/C++/Seleccion/Ejercicio6.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
	cout << "Convertir de Grados fahrenhed a celsius.\n";
	double Fahrenhed, Celsius;
	cout << "Ingresar a grados fahrenhed: ";
	cin >> Fahrenhed;
	if (Fahrenhed >= -459.69) {
		Celsius= (Fahrenhed - 32) * 5/9;
		cout << "Celsius = " << Celsius << "\n";
	} else {
		cout << "Error de grados";
	}
	return 0;
}

File: 1.FDP/2° Competencia/Java/Seleccion/Ejercicio6.java
import java.util.Scanner;	

public class Ejercicio6 {
	public static void main (String[] args){
		Scanner scan= new Scanner(System.in);
		//
		System.out.println("Calcular la convercion de grados fahrenhed.\n");
		double Fahrenhed, Convercion;
		System.out.println("Fahrenhed: ");
		Fahrenhed= scan.nextDouble();
		if (Fahrenhed > -459.69) {
			Convercion= (Fahrenhed - 32) * 5/9;
			System.out.println("Celsius= " + Convercion);
		} else {
			System.out.println("Error en los grados fahrenhed");
		}
		//
	}
}